dark1/memberavatarstatus/adm/style/acp_mas_review.html

```diff
@@ -0,0 +1,84 @@
+{% INCLUDE 'overall_header.html' %}
+{% INCLUDECSS '@dark1_memberavatarstatus/mas_acp_style.css' %}
+{% INCLUDEJS '@dark1_memberavatarstatus/mas_acp_script.js' %}
+
+<h1>{{ MAS_EXT_NAME }}</h1>
+<div>{{ lang('ACP_MAS_BY') }}&nbsp;{{ MAS_EXT_DEV }}</div>
+<hr style="margin: 5px 0px 0px 0px; padding: 5px 0px 0px 0px; border-top: 2px solid #888;" />
+<h2>{{ lang('ACP_MAS_MODE_REVIEW') }}</h2>
+
+<form id="acp_board" method="post" action="{{ U_ACTION }}">
+
+	<fieldset>
+		<legend>{{ lang('ACP_MAS_RV_AV_SET') }}</legend>
+		<dl>
+			<dt>
+				<label for="dark1_mas_rv_av">{{ lang('ACP_MAS_RV_AV_LB') }}{{ lang('COLON') }}</label>
+				<br />
+				<div>{{ lang('ACP_MAS_RV_AV_EXPLAIN') }}</div>
+			</dt>
+			<dd>
+				<label><input type="radio" class="radio" name="dark1_mas_rv_av" value="1" {% if MAS_RV_AVATAR %}checked="checked"{% endif %} id="dark1_mas_rv_av" /> {{ lang('YES') }}</label>
+				<label><input type="radio" class="radio" name="dark1_mas_rv_av" value="0" {% if not MAS_RV_AVATAR %}checked="checked"{% endif %} /> {{ lang('NO') }}</label>
+			</dd>
+		</dl>
+		<dl>
+			<dt>
+				<label for="dark1_mas_rv_av_sz">{{ lang('ACP_MAS_RV_AV_SZ_LB') }}{{ lang('COLON') }}</label>
+				<br />
+				<div>{{ lang('ACP_MAS_AV_SZ_SML_EXPLAIN') }}</div>
+			</dt>
+			<dd>
+				<input type="number" min="9" max="99" size="3" maxlength="3" step="1" name="dark1_mas_rv_av_sz" value="{{ MAS_RV_AV_SIZE }}" oninput="sync_avatar_size('dark1_mas_rv_av_sz','div_dark1_mas_rv_av_sz')" id="dark1_mas_rv_av_sz" />&nbsp;{{ lang('ACP_MAS_AV_SIZE_PX') }}
```

Comment:
Avoid `&nbsp;` inline but use a normal space.

Reply:
But , I prefer `&nbsp;` over ` ` , coz it's better than normal space.
Also , no one in extension validation team pointed that out...